### ISSUE TITLE
Thank you for your work, but I want to ask, how can I get the last two pictures in "README"?

### ISSUE THREAD

**Issue by 1553545365** [Song-Jingyu/PointPainting#2]
Thank you for your work, but I want to know that how can I get the last two pictures in "README"?
I mean "Accuracy for each object category and task difficulty/Average accuracy across different object category" 
thanks

**Comment by Song-Jingyu** [comment]
Hi,

You should be able to obtain the accuracy after training. You might notice there is a log file in /detector/output. You can read the average accuracy in that log file.

Best,
Jingyu

**Comment by 1553545365** [comment]
> Hi,
> 
> You should be able to obtain the accuracy after training. You might notice there is a log file in /detector/output. You can read the average accuracy in that log file.
> 
> Best,
> Jingyu

**Comment by 1553545365** [comment]
> > Hi,
> > You should be able to obtain the accuracy after training. You might notice there is a log file in /detector/output. You can read the average accuracy in that log file.
> > Best,
> > Jingyu

**Comment by 1553545365** [comment]
> Hi,
> 
> You should be able to obtain the accuracy after training. You might notice there is a log file in /detector/output. You can read the average accuracy in that log file.
> 
> Best,
> Jingyu

In fact, I've got the results of the test set, but I'm not sure if the results have been Pointpainted?
This is my test datapath:
detector
├── data
│   ├── kitti
│   │   │── ImageSets
│   │   │── training
│   │   │   ├── calib
│   │   │   ├── image_2
│   │   │   ├── image_3
│   │   │   ├── label_2
│   │   │   ├── velodyne
│   │   │   ├── planes
│   │   │   ├── painted_lidar
**│   │   │── testing
│   │   │   ├── calib
│   │   │   ├── image_2
│   │   │   ├── velodyne**

Do I need to run painting.py for the test set, before I running test.py?

**Comment by Song-Jingyu** [comment]
The same answer, the you may not need to paint test set because we (Painting and detector) only use train set and validation set, which is all contained in training folder.
